### ISSUE TITLE
Add Report menu is very small when it has no content

### ISSUE THREAD

**Issue by alxndrsn** [medic/medic-webapp#1488]
# Narrative

Clicking the _[ + Actions ]_ button on the _Reports_ actionbar reveals a very empty menu if no forms are available.

# Demonstrative screenshot

<img width="547" alt="screen shot 2015-10-28 at 18 19 49" src="https://cloud.githubusercontent.com/assets/191496/10796914/89630af4-7da0-11e5-8e27-41270c1d2f28.png">

# Ideal

One of:

* the button would be grayed out if no forms are available (can probably check  `$scope.formDefinitions.length`)
* say something helpful like _[None]_ in the list if there are none available

**Comment by garethbowen** [comment]
Nice catch.

**Comment by diannakane** [comment]
I think the bottom of page toast is fine, though I am curious if CHWs will see it. I sent mockups for one that shows up in the center of the page which may be more obvious.

I saw the toast after submitting an assessment, but when I completed a treatment follow up form, I received a blank page -- see below.
![screen shot 2015-10-30 at 11 01 23](https://cloud.githubusercontent.com/assets/1675870/10853880/cac0e204-7ef5-11e5-8508-7598dbbbdb74.png)
